### DISCUSSION TITLE
rest client插件使用 - WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#6] [General]
# rest client插件使用 - WxW

rest client 是 vscode 中类似 Postman 的 api 接口请求利器。

https://www.whbbit.cn/2023/01/03/rest-client%E6%8F%92%E4%BB%B6%E4%BD%BF%E7%94%A8/

**Comment by Whbbit1999** [comment]
记得 `Content-Type: application/json;charset=UTF-8` 不能添加 `;`

**Comment by ytx0521** [comment]
牛啊大佬
